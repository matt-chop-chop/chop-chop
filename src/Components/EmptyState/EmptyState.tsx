import { Flex, Text } from "@chakra-ui/react";
import React from "react";

const EmptyState = () => {
  return (
    <Flex
      alignItems="center"
      background="var(--chakra-colors-light-background)"
      border="2px solid var(--chakra-colors-light-text)"
      borderRadius={4}
      flexDirection="column"
      justifyContent="center"
      minHeight="250px"
      p={4}
      width="100%"
    >
      <Text textAlign="center" variant="body-bold">
        No matching recipes found
      </Text>
      <Text mt={2} variant="body">
        Try updating the selected parameters
      </Text>
    </Flex>
  );
};

export default EmptyState;
