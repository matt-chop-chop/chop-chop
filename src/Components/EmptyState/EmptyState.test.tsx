import { shallow } from "enzyme";
import React from "react";
import EmptyState from "./EmptyState";

describe("EmptyState", () => {
  it("is exported from EmptyState", () => {
    expect(EmptyState).toBeDefined();
  });

  it("renders as expected", () => {
    const wrapper = shallow(<EmptyState />);

    expect(wrapper).toMatchSnapshot();
  });
});
