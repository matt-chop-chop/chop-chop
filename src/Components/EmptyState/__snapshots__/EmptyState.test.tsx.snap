// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState renders as expected 1`] = `
<Flex
  alignItems="center"
  background="var(--chakra-colors-light-background)"
  border="2px solid var(--chakra-colors-light-text)"
  borderRadius={4}
  flexDirection="column"
  justifyContent="center"
  minHeight="250px"
  p={4}
  width="100%"
>
  <Text
    textAlign="center"
    variant="body-bold"
  >
    No matching recipes found
  </Text>
  <Text
    mt={2}
    variant="body"
  >
    Try updating the selected parameters
  </Text>
</Flex>
`;
